ui/page/account/account_page.go

```diff
@@ -0,0 +1,272 @@
+	"github.com/crypto-power/cryptopower/ui/load"
+	"github.com/crypto-power/cryptopower/ui/modal"
+	"github.com/crypto-power/cryptopower/ui/page/components"
+	"github.com/crypto-power/cryptopower/ui/utils"
+	"github.com/crypto-power/cryptopower/ui/values"
+)
+
+const AccountID = "Account"
+
+type (
+	C = layout.Context
+	D = layout.Dimensions
+)
+
+type Page struct {
+	*load.Load
+	// GenericPageModal defines methods such as ID() and OnAttachedToNavigator()
+	// that helps this Page satisfy the app.Page interface. It also defines
+	// helper methods for accessing the PageNavigator that displayed this page
+	// and the root WindowNavigator.
+	*app.GenericPageModal
+
+	wallet sharedW.Asset
+
+	container     *widget.List
+	addAccountBtn *cryptomaterial.Clickable
+	accountsList  *cryptomaterial.ClickableList
+	accounts      []*sharedW.Account
+
+	exchangeRate   float64
+	usdExchangeSet bool
+}
+
+func NewAccountPage(l *load.Load) *Page {
+	pg := &Page{
+		Load:             l,
+		GenericPageModal: app.NewGenericPageModal(AccountID),
+		container: &widget.List{
+			List: layout.List{Axis: layout.Vertical},
+		},
+		addAccountBtn: l.Theme.NewClickable(false),
+		accountsList:  l.Theme.NewClickableList(layout.Vertical),
+		wallet:        l.WL.SelectedWallet.Wallet,
+	}
+
+	pg.loadWalletAccount()
+
+	return pg
+}
+
+func (pg *Page) loadWalletAccount() {
+	walletAccounts := make([]*sharedW.Account, 0)
+	accounts, err := pg.wallet.GetAccountsRaw()
+	if err != nil {
+		log.Errorf("error retrieving wallet accounts: %v", err)
+		return
+	}
+
+	for _, acct := range accounts.Accounts {
+		if acct.Number == dcr.ImportedAccountNumber {
+			continue
+		}
+		walletAccounts = append(walletAccounts, acct)
+	}
+
+	pg.accounts = walletAccounts
+}
+
+// OnNavigatedTo is called when the page is about to be displayed and
+// may be used to initialize page features that are only relevant when
+// the page is displayed.
+// Part of the load.Page interface.
+func (pg *Page) OnNavigatedTo() {
+	pg.usdExchangeSet = false
+	if components.IsFetchExchangeRateAPIAllowed(pg.WL) {
+		pg.usdExchangeSet = pg.WL.AssetsManager.RateSource.Ready()
+		go func() {
+			rate, err := pg.Load.WL.FetchExchangeRate()
+			if err != nil {
+				log.Error(err)
+				return
+			}
+			pg.exchangeRate = rate
+			pg.ParentWindow().Reload()
+		}()
+	}
+}
+
+// Layout draws the page UI components into the provided layout context
+// to be eventually drawn on screen.
+// Part of the load.Page interface.
+func (pg *Page) Layout(gtx C) D {
+	return pg.Theme.List(pg.container).Layout(gtx, 1, func(gtx C, i int) D {
+		return pg.Theme.Card().Layout(gtx, func(gtx C) D {
+			return components.UniformHorizontalInset(values.MarginPadding16).Layout(gtx, func(gtx C) D {
+				return layout.Flex{Axis: layout.Vertical}.Layout(gtx,
+					layout.Rigid(pg.headerLayout),
+					layout.Rigid(func(gtx C) D {
+						return layout.Inset{Bottom: values.MarginPadding12}.Layout(gtx, func(gtx C) D {
```

Comment:
A bottom margin won't be needed here if you apply the recommendation above to give the card top and bottom padding.